devops/jobs/CreateSandboxCI.groovy

```diff
@@ -61,8 +61,6 @@ class CreateSandboxCI {
                   booleanParam('reconfigure',true)
                   booleanParam('recreate',false)
                   predefinedProp('name_tag','continuous-integration')
-                  booleanParam('xqueue',true)
-                  booleanParam('xserver',true)
```

Comment:
The daily one runs the from scratch which will definitely run these.

https://github.com/edx/configuration/blob/master/playbooks/edx-east/edx_continuous_integration.yml